core/src/main/java/com/gdx/game/screen/GameScreen.java

```diff
@@ -92,6 +99,31 @@ public GameScreen(GdxGame gdxGame, ResourceManager resourceManager) {
         player = EntityFactory.getInstance().getEntity(ProfileManager.getInstance().getProperty("playerCharacter", EntityFactory.EntityType.class));
         player.registerObserver(this);
 
+        //initialize controls
+        HashMap<String, String> jsonMap;
+
+        try {
+            jsonMap = json.fromJson(HashMap.class, Gdx.files.local(PARTIAL_CONTROLS_SETTINGS_PATH));
+
+            if (DEFAULT_CONTROLS.size() != jsonMap.size()){
+                throw new SerializationException("Not valid control map");
+            }
+
+        }catch (SerializationException se){
```

Comment:
instead of throwing an exception, maybe just check that the map is null or not and bind the player's control with the appropriate map.

Reply:
Initially I've used this approach, but the Gdx from Jason throw an exception that crash the application if the controls json is deleted. So I used the try catching approach